apps/explorer/src/ui/ObjectVideoImage.tsx

```diff
@@ -23,20 +30,127 @@ const imageStyles = cva(['z-0 flex-shrink-0 relative'], {
 	},
 	defaultVariants: {
 		disablePreview: false,
+		orientation: 'even',
 	},
+	compoundVariants: [
+		// orientation: even
+		{
+			variant: 'xs',
+			orientation: 'even',
+			className: 'w-8',
```

Comment:
Instead of doing all of these compound widths, could you instead use aspect ratios to accomplish this?

Reply:
There are a few issues with using `aspect-ratio` here
1. Aspect ratio only works if we define a fixed width, in these cases, we need them to be fixed height instead
2. Aspect ratio prop can only work on immediate parent of the `img` tag. We already wrap the `img` tag here https://github.com/MystenLabs/sui/blob/main/apps/explorer/src/ui/image/Image.tsx with a div. Moving the `aspect-ratio` prop in here is pretty difficult since `ObjectVideoImage` also has its defined width/height container.

I am open for some suggestions here, but I am a bit stuck on how to handle these cases.

CC: @Jordan-Mysten